apps/domain/lib/domain/telemetry.ex

```diff
@@ -90,43 +94,34 @@ defmodule Domain.Telemetry do
   end
 
   # How far back to count handshakes as an active device
-  @active_device_window 86_400
+  # @active_device_window 86_400
```

Comment:
Same re: above -- this is an important metric we want to keep. Probably want to leave a comment here so we don't forget.

Reply:
skipping this comment to reduce cognitive load